pkg/util/net.go

```diff
@@ -36,23 +35,17 @@ import (
 var previousGatewayIP string = ""
 
 func GetHostIP() (string, error) {
-	// Prefer OVN-K gateway IP if it is the CNI
-	gatewayIP, err := ovn.GetOVNGatewayIP()
```

Comment:
shall we remove the `GetOVNGatewayIP` in ovn.go if it is no longer used?

Reply:
Yes, I think we can remove it too.